servlet.py

```diff
@@ -104,6 +104,9 @@ class BaseHandler(tornado.web.RequestHandler):
     scaleMtLogs = False
     verbosity = 0
 
+    pairs_graph = {}
+    paths = {}
```

Comment:
Can you add a comment for each one that explains the semantics of the data structure? i.e. what are the keys and values.